### PR TITLE
Rename server name to tm.local

### PR DESCRIPTION
**Pull request by dominikhajduk**
Proposal to prevent issues on Chrome. 

Am I right that after changes like that we should remove container `docker rm ID` and start them once more?

### CODE REVIEW

File: docker/nginx/default.conf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 server {
-    server_name tm.dev;
+    server_name tm.local;
```
Comment (partikus) [inline docker/nginx/default.conf:2]: maybe would better to switch to self-signed SSL connection?

Comment (dominikhajduk) [reply docker/nginx/default.conf:2]: @partikus sure but as MVP this is enough I think and can be improved

**Review by tarnawski — APPROVED** [review]
Yes. Use `docker-compose rm --stop --force && docker-compose up​`

File: docker/nginx/default.conf (hunk 1 above)
Comment (partikus) [reply docker/nginx/default.conf:2]: @dominikhajduk ok, so please create a ticket with appropriate description

**Review by partikus — APPROVED** [review]

File: docker/nginx/default.conf (hunk 1 above)
Comment (myForksFiles) [reply docker/nginx/default.conf:2]: not really, i've tested not signed certs and chrome still doesn't cooperate

Comment (dominikhajduk) [reply docker/nginx/default.conf:2]: @myForksFiles co self signed do not work at all?

**Pull request merged**